src/index.html

```diff
@@ -11,12 +11,106 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo_logo"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
```

Comment:
```suggestion
    <header class="header">
      <a
        href="index.html"
        class="logo"
      >
        <img
          src="images/logo.png"
          alt="Moyo_logo"
          class="logo__img"
        />
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
            >
              APPLE
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="#"
              class="nav__link"
            >
              SAMSUNG
            </a>
          </li>
```

pls fix everywhere

and pay attention to this rule
![image](https://github.com/mate-academy/layout_moyo-header/assets/108538921/ee8a1215-7f53-4859-88f2-01d12cc9e72b)